ts/frontend/src/app/views/benchmark/benchmark.view.html

```diff
@@ -1,5 +1,5 @@
 <lib-content>
-  <app-site-heading class="bp-8 flex w-full" [title]="benchmark?.name" [logo]="customization?.content?.logo | publicResource"></app-site-heading>
+  <app-site-heading class="bp-8 flex w-full" [title]="benchmark?.name" [logo]="customization?.content?.logo | publicResource" [headerImage]="benchmark?.contents?.headerImage"></app-site-heading>

```

Comment:
If you have time, it would be nice to have a working example in the data SQL scripts.